shapetree.ttl

```diff
@@ -0,0 +1,325 @@
+  rdfs:label "References"@en .
+
+:matchesUriTemplate 
+  a owl:DatatypeProperty ;
+  rdfs:domain :ShapeTree ;
+  rdfs:range xsd:string ;
+  rdfs:comment "Describes a URI Template (RFC 6570) that governs the naming of resources of this ShapeTree"@en ;
+  rdfs:isDefinedBy <> ;
+  rdfs:label "Matches URI Template"@en .
+
+:validatedBy
+  a owl:ObjectProperty ;
+  rdfs:domain :ShapeTree ;
+  rdfs:comment "Links a ShapeTree to a Shape"@en ;
+  rdfs:isDefinedBy <> ;
+  rdfs:label "Validated By"@en .
+
+:supports
+  a owl:ObjectProperty ;
+  rdfs:domain :ShapeTree ;
+  rdfs:range :ShapeTree ;
+  rdfs:comment "Describes a relationship wherein a ShapeTree acts as a support for another, for purposes of indexing, aggregation, etc." ;
+  rdfs:isDefinedBy <> ;
+  rdfs:label "Supports"@en .
+
+#
+# Specific IRIs used by :contains predicate to define behavior 
+# when a matching ShapeTree with a URI Template is not found.
+# These values can be combined, for example,
+# :contains :AllowResources :AllowContainers :AllowNonRDFSources is equivalent to :AllowAll
+#
+:AllowResources
+  a :ShapeTree ;
+  rdfs:comment "If no matching :contains ShapeTree is found, allow if the requested content is a :ShapeTreeResource"@en ;
+  rdfs:isDefinedBy <> ;
+  rdfs:label "AllowResources"@en .
+
+:AllowContainers
+  a :ShapeTree ;
+  rdfs:comment "If no matching :contains ShapeTree is found, allow if the requested content is a :ShapeTreeContainer"@en ;
+  rdfs:isDefinedBy <> ;
+  rdfs:label "AllowContainers"@en .
+
+:AllowNonRDFSources
+  a :ShapeTree ;
+  rdfs:comment "If no matching :contains ShapeTree is found, only allow if the requested content is a :ShapeTreeNonRDFResource"@en ;
+  rdfs:isDefinedBy <> ;
+  rdfs:label "AllowNonRDFSources"@en .
+
+:AllowAll
+  a :ShapeTree ;
+  rdfs:comment "If no matching :contains ShapeTree is found, allow all types of unexpected content"@en ;
+  rdfs:isDefinedBy <> ;
+  rdfs:label "AllowAll"@en .
+
+:AllowNone
+  a :ShapeTree ;
+  rdfs:comment "If no matching :contains ShapeTree is found, do not allow any content"@en ;
+  rdfs:isDefinedBy <> ;
+  rdfs:label "AllowNone"@en .
+
+# ShapeTreeContainer IRI (maps to ldp:Container)
+#############################################################################
+# An IRI that represents a generalized container used in a ShapeTree
+#############################################################################
+:ShapeTreeContainer 
+  rdfs:comment "Describes a generalized container within a ShapeTree ecosystem"@en ;
+  rdfs:isDefinedBy <> ;
+  rdfs:label "ShapeTreeContainer"@en .
+
+
+# ShapeTreeResource IRI (maps to ldp:Resource)
+#############################################################################
+# An IRI that represents a generalized resource used in a ShapeTree
+#############################################################################
+:ShapeTreeResource 
+  rdfs:comment "Describes a generalized resource (non-container) within a ShapeTree ecosystem"@en ;
```

Comment:
comment may want to call it out as an rdf resource if everything else is a "NonRDFSource".

Reply:
Updated.